src/TimeWarpEventSet.cpp

```diff
@@ -84,9 +85,16 @@ void TimeWarpEventSet::insertEvent (unsigned int lp_id, std::shared_ptr<Event> e
 
         assert(input_queue_[lp_id]->size() == 1);
 
+
+#if defined (ONE_THREAD_PER_LTSF)
```

Comment:
instead of
#if defined(ONE_THREAD_PER_LTSF)
#else
    lock/unlock code
#endif
use the following format:
#if !defined(ONE_THREAD_PER_LTSF)
    lock/unlock code
#endif

Make this change everywhere.

Reply:
Also add this flag inside TimeWarpEventSet.hpp to deactivate the place schedule_queue_lock_ is declared in the event set class.

Reply:
Done